front/src/components/App/Maintenance/Maintenance.tsx

```diff
@@ -0,0 +1,16 @@
+import Button from "../../UI/Button/Button";
+import RouteLink from "../../UI/RouteLink/RouteLink.styled";
+import { APP_PAGE } from "../App";
+
+const Maintenance = () => {
```

Comment:
Jeszcze bym prosił żeby te komponenty nazwać według tej konwencji co SalesArea. Czyli w tym wypadku MaintenanceArea. To samo tyczy się pliku. Tak w celu zachowania spójności